docs/en/observability/profiling-troubleshooting.asciidoc

```diff
@@ -0,0 +1,220 @@
+Failed to setup gRPC connection (retrying...): context deadline exceeded
+----
++
+Verify the `collection-agent` configuration value is set and is equal to what was printed by `elastic-profiling config`.
+* The secret token is not valid, or it has been changed. In this case, the host-agent shuts down, and logs a similar message to the following:
++
+----
+rpc error: code = Unauthenticated desc = authentication failed
+----
+* The host-agent is unable to reach the `collectionagent` service. In this case, a similar message to the following is logged:
+----
+Failed to report hostinfo (retrying...): rpc error: code = Unimplemented desc = unknown service collectionagent.CollectionAgent"
+----
++
+This typically means that your {ecloud} cluster has not been configured for Universal Profiling. To confingure your {ecloud} cluster, follow the steps in <<profiling-configure-data-ingestion,configure data ingestion>>.
+* The APM server (part of the backend in {ecloud} that receives data from the host-agent) ran out of memory. In this case, a similar message to the following is logged:
+----
+Error: failed to invoke XXX(): Unavailable rpc error: code = Unavailable desc = unexpected HTTP status code received from server: 502 (Bad Gateway); transport: received unexpected content-type "application/json; charset=UTF-8"
+----
++
+Verify that the APM server is running by navigating to *{ecloud} → Deployments → `<Deployment Name>` → Integrations Server* in https://cloud.elastic.co/home[Elastic Cloud]. If the *Copy endpoint* link next to *APM* is grayed out,
+you need to restart the APM server by clicking *Force Restart* under *Integrations Server Management*.
++
+![restart-integrations-server](./img/restart-integrations-server.png)
++
+For non-demo workloads, verify that the Integrations Server has at least the recommended 4GB of RAM. You can check this on the Integrations Server page under *Instances*.
+
+If you're unable to find a solution to the host-agent failure, you can raise a support request indicating `Universal Profiling` and `host-agent` as the source of the problem.
+
+[discrete]
+[[profiling-enable-verbose-logging]]
+== Enable verbose logging in host-agent
+
+During the support process, you may be asked to provide debug logs from one of the host-agent installations from your
+deployment.
+
+To enable debug logs, add the `-verbose` command-line flag or the `verbose true` setting in the configuration file.
+
+NOTE: We recommend only enabling debug logs on a single instance of host-agent rather than an entire deployment to limit the amount of logs produced.
+
+[discrete]
+[[profiling-improve-load-time]]
+== Improve load times
+
+The amount of data loaded for the flamegraph, topN functions, and traces view can lead to latency when using a slow connection (e.g. DSL or mobile).
+
+If you're running Kibana/ES 8.6+, setting the Kibana cluster option `server.compression.brotli.enabled: true` reduces the amount of data transferred and should reduce load time.

```

Comment:
Given that this guide will only be available starting with 8.7.0 we can remove the statement about the minimum version of 8.6+?